include/antler/project/object.hpp

```diff
@@ -95,7 +95,11 @@ namespace antler::project {
          const auto& itr = m_dependencies.find(dep.name());
          bool has_value = itr == m_dependencies.end();
```

Comment:
sounds more logical and if you do this please update accordingly logic below.
```suggestion
         bool has_value = itr != m_dependencies.end();
```

Reply:
done